src/libs/antares/study/parts/thermal/cluster_list.cpp

```diff
@@ -405,6 +414,15 @@ bool ThermalClusterList::saveToFolder(const AnyString& folder) const
             if (!Math::Zero(c.variableomcost))
                 s->add("variableomcost", Math::Round(c.variableomcost,3));
 
+            // ramping
+            if (not Math::Zero(c.powerIncreaseCost))
+                s->add("power-increase-cost", Math::Round(c.powerIncreaseCost, 3));
+            if (not Math::Zero(c.powerDecreaseCost))
+                s->add("power-decrease-cost", Math::Round(c.powerDecreaseCost, 3));
+            if (not Math::Zero(c.maxUpwardPowerRampingRate))
+                s->add("max-upward-power-ramping-rate", Math::Round(c.maxUpwardPowerRampingRate, 3));
+            if (not Math::Zero(c.maxDownwardPowerRampingRate))
```

Comment:
`Math` is part of `Yuni`, which is deprecated. Please use [STL functions](https://en.cppreference.com/w/cpp/numeric/math/round) instead.